hadoop-processing/steps/01_logging/after-deploy

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+
+if [[ "$LOGGINGPLUGIN" == "rsyslog" ]]; then
+    ip=$(unitAddress rsyslog 0)
+    setResult "Rsyslog is now configured. Syslog events are aggregated on $ip."
+elif [[ "$LOGGINGPLUGIN" == "Graylog" ]]; then
```

Comment:
@battlemidget i'm winging it here on what the value will be.  Is this the plugin name from metadata.yaml? is it case sensitive?